pages/apis/graphql/cookbooks/pipelines.md

```diff
@@ -57,6 +57,17 @@ When setting pipeline steps using the API, you must pass in a string that Buildk
 > To avoid writing an entire YAML file in a single string, you can place a <code>pipeline.yml</code> file in a <code>.buildkite</code> directory at the root of your repo, and use the <code>pipeline upload</code> command in your pipeline steps to tell Buildkite where to find it. This means you only need the following:
 > <code>steps: { yaml: "steps:\n - command: \"buildkite-agent pipeline upload\"" }</code>
 
+### Slug creation conventions
+
+Pipeline slugs are generated based on the pipeline name you provide during the pipeline creation.
+The maximum allowed character length for a pipeline slug is `100`.
+The supported character format for slug generation is:
+`/\A[a-zA-Z0-9]+[a-zA-Z0-9\-]*\z/`.
```

Comment:
This looks like a regex. If so, what is the purpose of mentioning it here?

Can this regex be used to convert a pipeline name to its intended slug? If so, then this needs to be explained here, as this is not obvious.

Reply:
I'm assuming that this is, based on the reason for including it here.